resources/js/components/global/NavBar.vue

```diff
@@ -9,12 +9,12 @@
 		<b-collapse is-nav id="nav_collapse">
 			<ul class="navbar-nav mr-auto">
                 <li class="nav-item">
-					<router-link class="nav-link" to="/">Dashboard</router-link>
+					<router-link class="nav-link" to="/" :class="[dashboardActive ? 'active' : '']">Dashboard</router-link>
 				</li>
-				<li class="nav-item">
-					<router-link class="nav-link" to="/profile">My Profile</router-link>
+				<li v-if="user.role == 'student'" class="nav-item">
+					<router-link class="nav-link" :to="'/profile/' + userId" :class="[profileActive ? 'active' : '']">Profile</router-link>
```

Comment:
why not just store the profile information inside the vuex store? Routing to this page performs an api call but because of this load time for dynamic profiles are slow.